src/BrazeEndOfArticleComponent.ts

```diff
@@ -58,8 +58,6 @@ const END_OF_ARTICLE_MAPPINGS: ComponentMapping<
     [EPIC_NAME]: Epic,
     [EPIC_WITH_HEADER_IMAGE_NAME]: EpicWithSpecialHeader,
 
-    [NEWSLETTER_EPIC_NAME]: NewsletterEpic,
```

Comment:
Interestingly, I think this component (which the specific newsletter epic components use under the hood) has the same functionality & interface as the new `EpicNewsletter_SelfServe`. The difference is that the generic `NewsletterEpic` functionality wasn't ever exposed in storybook, but I think that's the only reason it wasn't/couldn't be used by marketing. I think we could combine the work you've done here to surface things in storybook with the existing `NewsletterEpic` component, rather than having two similar (identical?) components: `NewsletterEpic` and `EpicNewsletter_SelfServe`. So the same generic component can either be used directly by marketing, for full flexibility, and continue to be used by the specific newsletter epic components (e.g. `AUNewsletterEpic`) for the time being. At some point I think we should deprecate those and the generic component should be used in all cases.

Reply:
My initial attempt at this work was to create a `stories.tsx` file in the `NewsletterEpic` folder. It was by far the simplest solution ... but for the life of me I couldn't get the epic to display in Storybook.

This solution is uglier, but it works.

Thoughts?

Reply:
Ah right yeah that's what I was suggesting. Weird that you couldn't get it to display 🤔  I'm not sure why that would be but happy to pair if it'd be useful to try and figure it out?